Sources/ArcGISToolkit/Components/Augmented Reality/FlyoverSceneView.swift

```diff
@@ -18,9 +18,12 @@ import ArcGIS
 
 /// A scene view that provides an augmented reality fly over experience.
 @preconcurrency
+@available(visionOS, unavailable)
```

Comment:
I think this might have been missed by the Xcode 16 upgrade, but `preconcurrency` is no longer needed since `View` has that.

```suggestion
@available(visionOS, unavailable)
```

Reply:
Ok, fair enough. I noticed some views had that and others didn't but I didn't touch it because I thought it was deliberate. But if you are saying it is good then I'll make the change.

Reply:
It was needed when the `MainActor` attribute was added since that was adding a new concurrency constraint, but with the move to Xcode 16, that constraint is now on the `View` protocol, so neither `MainActor` nor `preconcurrency` is needed anymore.